util-images/network/netperfbenchmark/cmd/main.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/*
```

Comment:
by convention we generally don't use /* */ style comments for anything else than the copyright

Please switch to:

// Package ...

Reply:
BTW - this comment is useless - it's obvious what main is doing :)

Reply:
Removed the comment.